mantidproject/mantid
# Change ALC Tests To Run In Qt5

This is for **release-next** so that we can get the ALC tests running again as we have two fixes/changes for the ALC for the next release (#30458 and #30454)

**Description of work.**
The ALC tests were not being run in Qt5, this PR creates a new test target specific for the muon tests to be run in Qt5

**To test:**
1. Open VS
2. Check there is a new test target specific for muon interfaces tests
3. Check the ALC tests have been removed from the general scientific interfaces test and only appear in the new muon target
4. Check all tests pass (should do they have not been changed)

Fixes #30466 

*This does not require release notes* because **it is maintenance**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
